Makefile

```diff
@@ -56,22 +54,14 @@ lint:
 	pre-commit run --all-files --show-diff-on-failure
 
 test:
-	pytest --junitxml=test-results/junit.xml
+	pytest
 
 test-all:
 	tox
 
 test-integration: install
 	behave integration-tests/
 
-coverage-all:
-		coverage erase
-		coverage run --source sceptre -m pytest
-		coverage xml
-
-coverage: coverage-all
-		coverage report --show-missing --fail-under 92
-
 sonar:
```

Comment:
hmm, isn't sonar for aggregating coverage reports?  Do you know anything about sonar @tarkatronic?  I've never used it before.

Reply:
[Sonar](https://www.sonarqube.org/) is generally used for static analysis. I don't personally use it; I prefer using tools build specifically for and by the Python ecosystem including flake8, pylint, and maybe even bandit.

I wasn't sure I wanted to rip out a whole other part of the tooling yet, until I better understood how and why it's used.

Reply:
Agreed, Sonar's not that great with Python, it's much more useful for Java projects and C/C++ with a paid subscription. I feel we got what Sonar provides sufficiently with pre-commit linters. Unless someone wants a nice UI and historical view of the data.